contents/handbook/engineering/development-process.md

```diff
@@ -57,55 +30,112 @@ After the sprint planning, each team should have their own session (can be async
+
+### Evaluate success 
+
+Review impact of each major milestone and feedback into the planning process.
+
+When we review the status of goals we classify them as follows:
 
-### Types of big ambitious goals
+* **Nailed it:** We hit the goal spectacularly. High fives all round.
+* **Scraped it:** We _almost_ hit the goal, but we'll need to do a little bit more next sprint to tidy up. We should adjust our workload to have fewer resources on big goals during the next sprint to comfortable get this finished.
+* **Failed it:** We were nowhere near hitting the goal, but we learned some valuable lessons. We're going to go back to the drawing board. Maybe the goal wasn't right or maybe there's a different way to approach it?
 
-* **Moonshots:** These are big scary goals, we expect to fail to achieve them 50% of the time, if we fail we expect to learn something equally as valuable as if we succeed (just scraping the goal counts as a success)
-* **Roofshots:** These might also be big but we expect to achieve them 100% of the time, these can be goals where we cannot afford to fail (e.g. Launch feature to keep us compliant with new regulation) or where we are confident in our approach and don’t foresee unexpected risks / issues.
+## What about the small stuff?
 
-### Outcomes of goals
+Not everything directly contributes to a company level goal. It’s important that the small stuff also gets done for us to succeed. Use the following principles:
 
-When we review the status of the goals from the previous sprint, we'll classify them as follows:
-* **Nailed it:** We hit the goal spectacularly (high fives all round)
-* **Scraped it:** We almost hit the goal - but we'll need to do a little bit more next sprint to tidy up (we should adjust our workload to have fewer resources on big goals during the next sprint to comfortable get this finished)
-* **Failed it:** We were no-where near hitting the goal, but we learned some valuable lessons - we're going to go back to the drawing board - maybe the goal wasn't right or maybe there's a different way to approach it
+* **Yes, and**: Be encouraging and helpful with others who are innovating. All of our biggest wins have looked like bad ideas early on.
+* **Dogfooding**: Use the product yourself. When you see something that annoys you, fix it.
+* **Sidequests:** Smaller projects you are passionate about but may not shoot up our metrics (e.g. turbo mode).
+* **Support hero:** Support hero dedicates all of their time to customers, solving the wild and wonderful issues our customers find each week.
+
+## Sizing tasks and reducing WIP
+
+Efficient engineering organizations actively [reduce Work In Progress](https://loom.com/share/5efceb288b634a449041918bdba08202) (WIP) to avoid negative feedback loops that drive down productivity. 
 
-## 2. Sizing a task and reducing WIP
+Hence, a PR should be optimised for two things: 
```

Comment:
US not EU!! ;) 

optimised → optimized

Reply:
```suggestion
Hence, a PR should be optimized for two things: 
```